lib/worldwide/regions_loader.rb

```diff
@@ -46,6 +46,9 @@ def apply_hierarchy(parent:, code:, children:)
         @regions << current_region
       end
 
+      if !parent.nil? && current_region.parent != parent
```

Comment:
So the last reference in world.yml wins.  Makes sense to me. Also, nit: `if parent.present? && ...`